pkgs/misc/drivers/hplip/default.nix

```diff
@@ -58,6 +58,7 @@ pythonPackages.buildPythonApplication {
     sane-backends
     dbus
     net_snmp
+    nettools
```

Comment:
Does adding `netools` to `propagatedBuildInputs`  actually have the wanted effect?